src/content/en/2021/cms.md

```diff
@@ -159,11 +158,13 @@ Drilling into the adoption by CMS across all websites, out of 218 different CMS
 
 WordPress, the most commonly used platform, is used by 33.6% of these websites, up from 31.4% in 2020, a 7% increase in total adoption.
 
-Joomla and Drupal adoption is dropping - Joomla sites accounted for 1.9% of websites, down from 2.1% last year (9.5% decrease), and Drupal dropped from 2% to 1.8% (10% decrease).
+In percentage terms, Joomla and Drupal adoption is dropping–Joomla sites accounted for 1.9% of websites, down from 2.1% last year (9.5% decrease), and Drupal dropped from 2% to 1.8% (10% decrease). Absolute adoption did increase, but as a percentage of both overall CMS usage and of our (ever increasing!) data set, it is smaller.
+
+{# TODO Added a comment about absolute figures as think it's interesting. Feel free to remove if you disagree. #}
```

Comment:
@alonkochba you OK with this?

Reply:
I'm fine with this clarification, though it's a bit hard to determine if absolute adoption grew or not, since the dataset itself is growing every year.

Reply:
Yeah I agree. Just conscious that some of the CMSs might complain with the previous wording and argue differently. Think this clarification covers us a bit should that happen.

Reply:
```suggestion
In percentage terms, Joomla and Drupal adoption is dropping–Joomla sites accounted for 1.9% of websites, down from 2.1% last year (9.5% decrease), and Drupal dropped from 2% to 1.8% (10% decrease). Absolute adoption did increase, but as a percentage of both overall CMS usage and of our (ever increasing!) data set, it is smaller.
```